.eslintrc.js

```diff
@@ -42,7 +42,25 @@ module.exports = {
 				plugins: [ 'jsx' ],
 			},
 		},
+
 	},
+	overrides: [
+		{
+			files: [ '*.ts', '*.tsx' ],
+			extends: [
+				'plugin:@typescript-eslint/recommended',
+				// 'plugin:@typescript-eslint/recommended-requiring-type-checking',
+			],
+			rules: {
+				'@typescript-eslint/await-thenable': 2,
+				'@typescript-eslint/no-var-requires': 2,
+				'@typescript-eslint/ban-ts-comment': 2,
+			},
```

Comment:
1. If we go with "strict", you can probably remove those since those are enabled there
2. if we keep it, please use "error" instead of 2, to make it clearer

Reply:
1. If I enable strict now, we will have 500-1000 new errors. Lets do it one by one in next PR.
2. Fixed.